client/components/map.jsx

```diff
@@ -1,39 +1,86 @@
 import React from 'react';
-import {
-  GoogleMap,
-  LoadScript,
-  Marker
-} from '@react-google-maps/api';
-import AppContext from '../lib/app-context.js';
+import { GoogleMap, LoadScript, Marker } from '@react-google-maps/api';
+
 const id = ['c3caed1a16123b9f'];
 const key = 'AIzaSyCn2kPCBIEv3Tkz2pI-W88ra2TeqHqdxfo';
```

Comment:
Your maps key is now leaked 😢. Be sure to get new ones and put them in your `.env` file.

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#envexample-and-env